homeassistant/components/progettihwsw/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/progettihwsw",
   "iot_class": "local_polling",
   "loggers": ["ProgettiHWSW"],
-  "requirements": ["progettihwsw==0.1.1"]
+  "requirements": ["progettihwsw==0.1.3"]
```

Comment:
Please tag github releases corresponding to the PyPI releases to make it easier for the community to track changes.

Reply:
@ArdaSeremet please add tag on https://github.com/ArdaSeremet/progettihwsw for version 0.1.3, actualy if present for 0.1.1. thanks so much

Reply:
@ArdaSeremet there is update for tag creation???

Reply:
Created the tag 0.1.3 on Github repo.